lib/hiera/config.rb

```diff
@@ -54,8 +54,7 @@ def validate!
         begin
           require "deep_merge"
         rescue LoadError
-          Hiera.error "Must have 'deep_merge' gem installed for the configured merge_behavior."
-          raise
+          raise "Must have 'deep_merge' gem installed for the configured merge_behavior."
```

Comment:
this raises a RuntimeError, not a LoadError which is expected by the rspec tests. Should either raise a new LoadError, or a hiera specific error (there is one for configuration errors).